templates/arm/omap-qt/mods.config

```diff
@@ -1,10 +1,12 @@
 package genconfig
 
 configuration conf {
+	@Runlevel(0) include embox.arch.arm.armlib.high_excpt_table
 	@Runlevel(0) include embox.arch.generic.arch
-	@Runlevel(0) include embox.arch.arm.mmu_test
+	//@Runlevel(0) include embox.arch.arm.mmu_test
```

Comment:
If this template without mmu then we can simply remove mmu_test, there is no need to comment it out.